collections/_get-started/installation.html

```diff
@@ -3,65 +3,74 @@
 layout: layout-sidenav
 description: Installation guide that will help you quicky spin up a site using SGDS
 ---
+{% capture using-node %}
+npm i @govtechsg/sgds bootstrap-icons
+{% endcapture %}
+
 {% capture reactjs %}
 npm install @govtechsg/sgds-react
 
-//not required if using cdn
+// not required if using cdn
 npm install @govtechsg/sgds-govtech bootstrap-icons
 {% endcapture %}
-{% capture using-node %}
-npm i @govtechsg/sgds bootstrap-icons
```

Comment:
should be npm install **@govtechsg/sgds** bootstrap-icons 

not, npm install @govtechsg/sgds-govtech bootstrap icons

Reply:
resolved in commit [c4e7d4f](https://github.com/GovTechSG/sgds/pull/372/commits/c4e7d4f332ef6c0e50a7429b0a143a05ad8981ba)